parts/kubernetesagentresourcesvmas.t

```diff
@@ -15,20 +15,26 @@
       "name": "[concat(variables('{{.Name}}VMNamePrefix'), 'nic-', copyIndex(variables('{{.Name}}Offset')))]",
       "properties": {
 {{if .IsCustomVNET}}
-	    "networkSecurityGroup": {
-		    "id": "[variables('nsgID')]"
-	    },
+        "networkSecurityGroup": {
+          "id": "[variables('nsgID')]"
+        },
 {{end}}
         "ipConfigurations": [
+          {{range $seq := loop 1 .IPAddressCount}}
           {
-            "name": "ipconfig1",
+            "name": "ipconfig{{$seq}}",
             "properties": {
+              {{if eq $seq 1}}
+              "primary": true,
+              {{end}}
               "privateIPAllocationMethod": "Dynamic",
               "subnet": {
-                "id": "[variables('{{.Name}}VnetSubnetID')]"
-             }
+                "id": "[variables('{{$.Name}}VnetSubnetID')]"
```

Comment:
Does this have different semantic meaning? `$.Name` vs `.Name`? If not, can you revert the change to this line for consistency's sake?